build.sh

```diff
@@ -255,23 +255,25 @@ build_gmake() {
     cp make gmake
 }
 
-JDK_DIR=openjdk-8u72b15-bsd-port-20160220
+# We use the JDK we just built for all consequent Java compilation (and as a
+# basis for the Graal compiler).
 case `uname` in
-    Linux)
-      JDK_JAVAC=${wrkdir}/openjdk/build/linux-x86_64-normal-server-release/jdk/bin/javac;;
-    OpenBSD)
-      JDK_JAVAC=${wrkdir}/openjdk/build/bsd-x86_64-normal-server-release/jdk/bin/javac;;
+    Linux)   OUR_JAVA_HOME=${wrkdir}/openjdk/build/linux-x86_64-normal-server-release/images/j2sdk-image/;;
```

Comment:
I don't think so. I replaced the variable that points to a javac with one which points to a JAVA_HOME, since part of the build wants me to specify a JAVA_HOME and javac is easily expressed in terms of a JAVA_HOME.